File: 프로그래머스/1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.py
def solution(n, arr1, arr2):
    answer = []
    
    for i in range(len(arr1)):
        value = ''
        temp1 = list(bin(arr1[i])[2:].zfill(n))
        temp2 = list(bin(arr2[i])[2:].zfill(n))
        for j in range(n):
            if temp1[j] == '1' or temp2[j] == '1':
                value += "#"
            else:
                value += " "
        answer.append(value)
    return answer

File: 프로그래머스/1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.js
function solution(n, arr1, arr2) {
    let answer = Array(n).fill("");
    
    for(let i=0; i<n; i++){
        let temp1 = arr1[i].toString(2).padStart(n,'0');
        let temp2 = arr2[i].toString(2).padStart(n,'0');
        
        for(let j=0; j<n; j++){
            if(temp1[j] == '1' || temp2[j] == '1'){
                answer[i] += "#";
            }
            else if(temp1[j] == '0' && temp2[j] == '0'){
                answer[i] += " ";
            }
        }
    }
    return answer;
}